data/content_scripts/posting_process/context_menu.js

```diff
@@ -35,7 +35,7 @@ postingProcess.menuClickHandler = function(node, data) {
     text: selectionText,
     pageURL: window.location.href,
   };
-  self.postMessage(info);
+  self.postMessage({name: "startPostingProcess", content: info});
```

Comment:
In Chrome extension I use Chrome native interface to send context menu messages as well :-)